tests/integration/ceph_helm_test.go

```diff
@@ -102,3 +102,61 @@ func (hs *HelmSuite) TestZFileStoreOnRookInstalledViaHelm() {
 func (hs *HelmSuite) TestObjectStoreOnRookInstalledViaHelm() {
 	runObjectE2ETestLite(hs.helper, hs.kh, hs.Suite, hs.namespace, "default", 3)
 }
+
+func TestCephHelmSuiteOnPVC(t *testing.T) {
```

Comment:
Instead of creating a new suite, I'd suggest modifying the existing helm suite. Two conditions need to be met in order to run the tests on PVCs:
- The k8s version is >= 1.13
- The `gp2` storage class exists in the cluster. (Our CI cluster might not have this storage class, we'll need to make sure it is there)

If those conditions are met, pass `true`, otherwise pass `false`. Then the helm test can run even if PVCs aren't available.

Reply:
done